package.json

```diff
@@ -15,17 +15,17 @@
     "react-confetti": "^6.1.0",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.0",
     "web-vitals": "^2.1.2"
   },
+  "type": "module",
   "scripts": {
-    "predeploy": "npm run build && cp CNAME build/CNAME",
+    "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
+    "start": "vite",
+    "preview": "vite preview",
     "start:local-docker": "docker container run --rm -it -p 3000:3000 developer-quiz-site",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "build": "vite build",
+    "test": "vitest",
```

Comment:
when I ran the test command, all the tests pass fine but it generates a results.json file

<img width="340" alt="Screenshot 2023-12-22 at 8 18 44 AM" src="https://github.com/freeCodeCamp/Developer_Quiz_Site/assets/67210629/c9981bd5-7316-47c4-a1a8-f088a3017fc6">

It would be best to add the file to the gitignore so it doesn't get committed to the repo by accident.

Reply:
configured tests to output to tests_results folder